src/components/pages/data/list/use-response-list.tsx

```diff
@@ -47,25 +55,29 @@ export const ResponseListProvider = ({
   const [map, setMap] = useState([]);
   const { lang } = useTranslation();
   const [isLoading, setIsLoading] = useState(false);
+  const [list_results, setListResults] = useState([]);
+  const [map_results, setMapResults] = useState([]);
+  const [isSearching, setIsSearching] = useState(false);
+  const [query, setQuery] = useState("");
 
   const fetchListData = async () => {
     NProgress.start();
     setIsLoading(true);
-
-    const payload = { ...filter.f, language: lang, limit: LIST_PAGINATION_LIMIT };
-
+    const payload = { ...filter.f, language: lang, limit: LIST_PAGINATION_LIMIT, query };
+    const response = await axSearchCCAData(payload);
```

Comment:
rename to something specific like ccaSearchDataResponse

Reply:
renamed to ccaDataResponse